02_calculator/calculator.rb

```diff
@@ -0,0 +1,51 @@
+def add(x, y)
+  x+y
+end
+
+def subtract(x, y)
+  x-y
+end
+
+def sum(a = Array)
+  sumVal = 0;
+  for i in 0..a.size-1
```

Comment:
Can you find something useful in https://ruby-doc.org/core-3.0.1/Enumerable.html that may help you do this without for / while. Ruby was designed to be read as close to english as possible.